File: Animal/Animal.java
class M{
	public M(){
		Animal a = new Animal("dog","woof");
		a.getCry();
	}
}
public class Animal{
	private String type;
	private String cry;
	public Animal(String type,String cry){
		this.type = type;
		this.cry = cry;
	}
	public String getType(){
		return this.type;
	}
	public String getCry(){
		return this.cry;
	}
}

File: Animal/Animal.py
class M:
    def __init__(self):
        a = Animal('dog', 'woof')
        getCry()

class Animal:
    def __init__(self, type, cry):
        self.type = type
        self.cry = cry

    def getType(self):
        return self.type

    def getCry(self):
        return self.cry

